runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -43,18 +45,182 @@
 @AllArgsConstructor
 public class CoordinateClue extends ClueScroll implements TextClueScroll, LocationClueScroll
 {
+	public static final Map<WorldPoint, String> CLUES = new HashMap<WorldPoint, String>()
```

Comment:
Use `ImmutableMap::builder`

Reply:
also make it private

Reply:
Can you tell me if its the right way of doing ImmutableMap.
ty

Reply:
Looks right to me. :+1: